pages/SameSite.md

```diff
@@ -21,17 +21,18 @@ A bank website however most likely doesn't want to allow any transactional pages
 
 The `lax` value provides a reasonable balance between security and usability for websites that want to maintain user's logged-in session 
 after the user arrives from an external link. In the above GitHub scenario, the session cookie would be allowed when following a regular link 
-from an external website while blocking it in CSRF-prone request methods (e.g. POST).
+from an external website while blocking it in CSRF-prone request methods (e.g. POST).  `lax` is as of December 2024 the default for Chrome, Edge and Opera but not for any Firefox or Safari. 
```

Comment:
![image](https://github.com/user-attachments/assets/0d085f5d-ae8c-44f4-a10d-c7fdf2d4efb5)

Reply:
The AI is halluzinating. I´d rather use hard facts, see the [added reference](https://caniuse.com/mdn-http_headers_set-cookie_samesite_lax_default)

Reply:
- https://hacks.mozilla.org/2020/08/changes-to-samesite-cookie-behavior/
- https://help.salesforce.com/s/articleView?id=000389944&type=1
- https://bugzilla.mozilla.org/show_bug.cgi?id=1617609 (marked fixed as of FF 96)
- https://github.com/mdn/content/issues/10857

- https://github.com/mdn/content/issues/36931